pkgs/top-level/all-packages.nix

```diff
@@ -12291,6 +12291,11 @@ let
 
   koji = callPackage ../tools/package-management/koji { };
 
+  konversation_1_6 = callPackage ../applications/networking/irc/konversation/1.6.nix {
+      makeQtWrapper = qt55.makeQtWrapper;
```

Comment:
You shouldn't import things from `qt55`; `kde5PackagesFun` will ensure that your package is called with the correct versions of these packages.